README.md

```diff
@@ -1 +1,80 @@
-# testing-practice
\ No newline at end of file
+# Testing Practice [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/MFahadUmer/todolist)
+
+[![License](https://img.shields.io/badge/License-MIT-green.svg)]()
+![GitHub followers](https://img.shields.io/github/followers/imhilla?label=imhilla&style=social)
+![Twitter URL](https://img.shields.io/twitter/follow/hillarykiptoo_?label=Follow&style=social)
+
+[![License](https://img.shields.io/badge/License-MIT-green.svg)]()
+![GitHub followers](https://img.shields.io/github/followers/MFahadUmer?label=mfahadumer&style=social)
+![Twitter URL](https://img.shields.io/twitter/follow/engfahadumer?label=Follow&style=social)
+
+> This is a Microverse Javascript Project
+
+> Testing Practice!
+
+## Features
+
+- Calculator 
+- Cipher Encryption
+- Array Analyzer
+- Capitalize
+
+In this project, the following features were done;
+
+### Functionalities added
+- Ability to test calculator  functions.
+- Ability to test Cipher encrytion.
+- Ability to test Array Analyzer.
+- Ability to test capitalize of string.
+
+## Built with
+
+- HTML
+- Git
+- CSS
+- JavaScript
+- Shields.io
+- Webpack
+- npm
+- command-line tools such as git.
+
+## Test with
```

Comment:
- [ ] This is not enough to run the tests locally, make sure to add instructions on downloading the project and installing the packages with npm before it.
Be as detailed and specific as possible in the README instructions, avoid assuming your user knows how to run the code/tests.